src/native/fs/fs_napi.cpp

```diff
@@ -650,11 +650,15 @@ struct FSWorker : public Napi::AsyncWorker
     {
         Napi::Env env = Env();
         DBG1("FS::FSWorker::OnError: " << _desc << " " << DVAL(error.Message()));
+        DBG0("FS::FSWorker::OnError: " << _desc << " " << DVAL(error.Message()));
```

Comment:
This might add a lot of log prints, is it needed to show the error desc? isn't the context field enough?

Reply:
The reason I added it was to be able to print all of the errors that we encounter on the napi layer. Not all the failures directly lead to a failure of a S3 request which means these can get lost. Having that said, it's a little bit between "debug" logging and "info" logging.

WDYT? Should I change it back to `DBG1`?

Reply:
dilemma indeed, but I think we cannot afford to always log this by default because this is something that happens often in normal flows. so unless you see that this is not the case, I would revert it to DBG1.

Reply:
Reverted log level.